app/src/test/kotlin/com/exallium/todoapp/allnotes/AllNotesPresenterTest.kt

```diff
@@ -79,11 +89,18 @@ class AllNotesPresenterTest {
         val publisher = PublishSubject.create<Unit>()
         `when`(view.addNoteClicks()).thenReturn(publisher)
         testSubject.onViewCreated()
+        val bundle: Bundle = given_bundle()
 
         // WHEN
         publisher.onNext(Unit)
 
         // THEN
-        verify(view).showSingleNote()
+        verify(view).showSingleNote(bundle)
+    }
+
+    private fun given_bundle() : Bundle {
+        val bundle: Bundle = mock(Bundle::class.java)
+        `when`(bundleFactory.createBundle()).thenReturn(bundle)
```

Comment:
Now that mockito-kotlin is in the app, you can use `whenever` instead of `when` for slightly nicer looking code